modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManagerToolbar.vue

```diff
@@ -146,8 +151,11 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+  .apos-pieces-manager-toolbar {
+    max-height: 37px;
```

Comment:
The problem is this:
```
.apos-schema[data-v-4d5681c9] .apos-field {
    margin-bottom: 30px;
    letter-spacing: 0.5px;
}
```

Because the manager is opening inside the schema component of a piece this is getting applied to the search field. Let's disable that bottom margin when inside the toolbar instead.

Reply:
I don't see the relationship between the field's height and the button. But it works. At least when you disable it through the browser. But you have the information you are in a toolbar much deeper than the field. 
I tried to remove the "scope" but no success. I will need a CSS expert on that one :grimacing:

Reply:
I pushed an update to this branch to fix this. You'll need to `git pull` before doing additional work on it.

Reply:
Noice